_posts/2021-05-14-bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton.md

```diff
@@ -0,0 +1,612 @@
+---
+title: "Bring GitOps to your OpenFaaS functions with Argo CD - The missing piece of the pipeline: Tekton"
```

Comment:
This title has a lot going on, and I'm not sure that people will click on it if they saw it in a newsletter.

Can we take a more practical approach of problem/benefit? Isn't this blog post about CI?

I'd prefer something like:

> Learn how to build your OpenFaaS Functions with Tekton.